common/persistence/elasticsearch/es_visibility_metric_clients.go

```diff
@@ -379,7 +379,7 @@ func (p *visibilityMetricsClient) updateErrorMetric(scopeWithDomainTag metrics.S
 		scopeWithDomainTag.IncCounter(metrics.ElasticsearchErrBusyCounterPerDomain)
 		scopeWithDomainTag.IncCounter(metrics.ElasticsearchFailuresPerDomain)
 	default:
-		p.logger.Error("Operation failed with internal error.", tag.MetricScope(scope), tag.Error(err))
+		p.logger.Error("Operation failed with internal error.", tag.MetricScope(int(scope)), tag.Error(err))
```

Comment:
there are a few of these:
for some reason we're logging the scope as
```json
{
  "metrics-scope": 123
}
```
which does not seem useful.  this should probably be pulling out the operation name (which is only possible due to #7327 making them unique)